Add Fastlane metadata for F-Droid (#15411)
minetest/minetest: fastlane/metadata/android/en-US/full_description.txt

Luanti is a voxel-based game platform with multiplayer capabilities.

The objective in the games available varies, from sandbox games where you can explore, dig and build in a very large voxel world, to arcade style games made with the engine.

There are a wide range of built-in map generators for procedually generated terrain, and voxel-based lighting that adjusts after the environment and light sources.

All games and mods are written in Lua using the scripting API which is easy to use and get started with for modders.

> Luanti is available for Windows, macOS, GNU/Linux, FreeBSD, OpenBSD, DragonFly BSD, and Android.

Reviewer: also the "servers" aspect is too small here imo

> Luanti is available for Windows, macOS, GNU/Linux, FreeBSD, OpenBSD, DragonFly BSD, and Android.

Reviewer 2: :+1: on grorp's suggestions

The focus of the description generally seems a bit arbitrary to me ..? For example the mention of three different BSD versions (which I'm not even so sure work terribly well, the ports are not done by the core team) while just summarizing Linux distros as "GNU/Linux", or the mention of the "voxel-based lighting system".

I would say that things like multiplayer, possibility of PvP, mobs etc. should be emphasized more.

That said I don't want to bikeshed this to death. This description is still a significant improvement and can be improved further in the future.

> Luanti is available for Windows, macOS, GNU/Linux, FreeBSD, OpenBSD, DragonFly BSD, and Android.

Reviewer 3: we should drop the "Luanti is available for" sentence, it's not relevant for Android at all.

> Luanti is available for Windows, macOS, GNU/Linux, FreeBSD, OpenBSD, DragonFly BSD, and Android.

Reviewer 4: :+1: on grorp's suggestions. The last sentence is not really needed, and technically incorrect for GNU/Linux. Luanti also runs on other libc implementations.

> Luanti is a voxel-based game platform with multiplayer capabilities.
> 
> The objective in the games available varies, from sandbox games where you can explore, dig and build in a very large voxel world, to arcade style games made with the engine.
> 
> There are a wide range of built-in map generators for procedually generated terrain, and voxel-based lighting that adjusts after the environment and light sources.
> 
> All games and mods are written in Lua using the scripting API which is easy to use and get started with for modders.
> 
> Luanti is available for Windows, macOS, GNU/Linux, FreeBSD, OpenBSD, DragonFly BSD, and Android.

Reviewer 3: (copying to apply)
suggestions:

```suggestion
Luanti is a voxel-based game platform with multiplayer capabilities.

The objective of the available games varies, from sandbox games where you can explore, dig and build in a very large voxel world, to arcade-style games.

There is a wide range of built-in map generators for procedurally generated terrain, and voxel-based lighting that adapts to the environment and light sources.

All games and mods are written in Lua with a scripting API that is easy to use and makes it easy for modders to get started.
```
